src/big/big_float.cr

```diff
@@ -559,21 +567,3 @@ module Math
     BigFloat.new { |mpf| LibGMP.mpf_sqrt(mpf, value) }
   end
 end
-
-# :nodoc:
-struct Crystal::Hasher
-  def self.reduce_num(value : BigFloat)
```

Comment:
**question:** How is this removal related to the other changes?

Reply:
I see. And with its removal, calls fall back to `reduce_num(value : Float)` which uses `Math.frexp`.